src/main/scala/com/rbmhtechnology/calliope/serializer/SequencedEventSerializer.scala

```diff
@@ -61,10 +64,14 @@ abstract class SequencedEventSerializer(system: ExtendedActorSystem, payloadSeri
       .setCreationTimestamp(sequencedEvent.creationTimestamp.toEpochMilli)
 
   def sequencedEvent(format: SequencedEventFormat): SequencedEvent[_] =
-    SequencedEvent(
-      payloadSerializer.payload(format.getPayload),
-      format.getSourceId,
-      format.getSequenceNr,
-      Instant.ofEpochMilli(format.getCreationTimestamp)
-    )
+    try {
+      SequencedEvent(
+        payloadSerializer.payload(format.getPayload),
+        format.getSourceId,
+        format.getSequenceNr,
+        Instant.ofEpochMilli(format.getCreationTimestamp)
+      )
+    } catch {
+      case NonFatal(err) => throw new PayloadNotSerializableException("Unable to deserialize payload", err, format.getPayload)
```

Comment:
Why to try/catch/throw here again? `PayloadSerializer.payload` already does that.